blocks/commerce-checkout/commerce-checkout.js

```diff
@@ -187,6 +187,31 @@ export default async function decorate(block) {
 
   block.appendChild(checkoutFragment);
 
+  const authPrivacyPolicyConsentSlot = {
+    PrivacyPolicyConsent: async (ctx) => {
+      const wrapper = document.createElement('span');
+      Object.assign(wrapper.style, {
+        color: 'var(--color-neutral-700)',
+        font: 'var(--type-details-caption-2-font)',
+        display: 'block',
+        marginBottom: 'var(--spacing-medium)',
+      });
+
+      const link = document.createElement('a');
+      link.href = '/privacy-policy';
```

Comment:
These are defined in `scripts/constants.js`

Reply:
Moved to `scripts/constants.js`